solarkraft/src/environment.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license
+ * [Apache-2.0](https://github.com/freespek/solarkraft/blob/main/LICENSE)
+ */
+declare global {
+    namespace NodeJS {
+        interface ProcessEnv {
+            APALACHE_HOME: string
+        }
+    }
+}
+
+export {}
```

Comment:
why do we need that? Is it some kind of an idiom for the case of running outside of node?

Reply:
It's used to specify the `process.env` object's `APALACHE_HOME` field.

Reply:
Cool. I did not know about that!